src/Entities/Request/SMSMessage.php

```diff
@@ -29,7 +29,10 @@ class SMSMessage implements JsonSerializable
     const CLASS_PREMIUM  = 'premium';
     const CLASS_SECRET   = 'secret';
 
-    private $message, $sender, $recipients, $tags, $sendtime, $class, $userref, $callbackUrl;
+    const ENCODING_GSM_7   = 'UTF8';
```

Comment:
I think we should call these what they contain. The fact that GatewayAPI uses GSM7 for UTF8 when we do this is probably something to link to instead. I don't know why they designed it like this, but I would prefer that the name of the constants reflect the _contents_ of the constant, i.e.

```php
const ENCODING_UTF8  = 'UTF8';
const ENCODING_UCS2  = 'UCS2';
```

If we didn't have PHP 7.3 support, I would change this to an enum, but that's going to be in next major.